### ISSUE TITLE
Specifying directory in loadSuites does not work

### ISSUE THREAD

**Issue by priyabrat1801** [Polymer/web-component-tester#226]
In my index file when i specify the directory where all my test files reside, wct could not find the directory.

I did 

    WCT.loadSuites(['polymer/']);
    Error: Unknown resource type: polymer/

    WCT.loadSuites(['polymer/*.html']);
    Error: Cannot GET /components/apm-application/test/polymer/*.html? 

    WCT.loadSuites(['polymer/**/*.html']);
    Error: Cannot GET /components/apm-application/test/polymer/**/*.html?

It does not work unless i provide individual file names in the loadSuites. Any work around for this or i am missing something.

**Comment by priyabrat1801** [comment]
Note that this is through the wct.conf.js file. If i pass the directory path in command line, it works.

**Comment by justinfagnani** [comment]
Yes, looks like loadSuites() does not support globs. In general, it'd be nice to run wct on patterns like this.

**Comment by Eugenitals** [comment]
+1
